recipes/dggrid/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dggrid" %}
+{% set version = "7.7" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sahrk/DGGRID/archive/v{{ version}}.tar.gz
+  sha256: 1dd74919798b393146cfc96cb6969d30660e4e260707426e6754b46b5de81a2a
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - cmake >=3.12,<4
+    - make  # [linux]
+  host:
+    - libgdal
+    - shapelib
+  run:
+    - libgdal
+    - shapelib
+
+test:
+  commands:
+    - test -f ${PREFIX}/bin/dggrid 
+    - if not exist %LIBRARY_BIN%\dggrid exit 1  # [win]
```

Comment:
Does ddgrid have a `--version` or `--help` command that you can run to check that the binary is executable in the conda environment?

Reply:
Not currently. The upstream author is informed and said he will add this at some point.
I can offer a minimal test run config:

```shell
echo "dggrid_operation GENERATE_GRID
dggs_type ISEA7H
dggs_res_spec 3
clip_subset_type WHOLE_EARTH
cell_output_type GDAL
cell_output_gdal_format GPKG
cell_output_file_name minout.gpkg" > min.meta && dggrid min.meta
```

This should return ok.

Reply:
Should this be like a separate script, or can it be added like that to the yaml?

Reply:
You can add do either.